src/components/ReportActionItem/ExportWithDropdownMenu.tsx

```diff
@@ -74,7 +74,7 @@ function ExportWithDropdownMenu({
             },
             {
                 value: CONST.REPORT.EXPORT_OPTIONS.MARK_AS_EXPORTED,
-                text: translate('workspace.common.markAsExported'),
+                text: translate('workspace.common.markAsEntered'),
```

Comment:
Is this intended?

Reply:
Yes, because `markAsExported` had value `Mark as manually entered`. So I updated the name to match its value and added new entry.